README.md

```diff
@@ -39,6 +39,60 @@ client.locations.list().then(locations => {
 
 ```
 
+## Logging
+
+There is some logging done of requests and responses made to the API. The
+default logger does nothing but you can pass your own. Logging is done via a generic interface so
+you can use whatever logger you want in your application.
+
+First, write an implementation of the `Logger` interface defined in
+[logger.ts](https://github.com/SmartThingsCommunity/smartthings-core-sdk/blob/master/src/logger.ts)
+which proxies to your logger. For example:
+
+```javascript
+import { Logger as WinstonLogger } from 'winston'
+
+
+export class WinstonLoggerProxy implements Logger {
```

Comment:
We should include the import for our `Logger` since it is undefined here.

Reply:
Good call. Fixed.